app/views/DataEntryView.js

```diff
@@ -0,0 +1,40 @@
+
+var Template = require('../util/Template');
+var TelemetryData = require('../models/TelemetryData')
+var fs = require('fs');
+var Logger = require('../util/Logger');
+var app_config=require('./config/application_config.js');
+
+
+module.exports = function (Marionette) {
+
+    return Marionette.ItemView.extend({
+        template: Template('DataEntryView'), 
+        className: 'gainsAdjustView', 
+
```

Comment:
Create an initialize function for this view, and in it add ```//create the logs directory if it doesnt yet exist
mkdirp.sync('./'+app_config.get('log_dir')+'data_entires/');``` to create the folder so we cant get an ambigious crash if that folder doesnt exist yet. Remember to add a `var mkdirp = require('mkdirp');` up top ^^^ first